staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -1218,6 +1238,7 @@ type LabelSelector struct {
 	MatchLabels map[string]string `json:"matchLabels,omitempty" protobuf:"bytes,1,rep,name=matchLabels"`
 	// matchExpressions is a list of label selector requirements. The requirements are ANDed.
 	// +optional
+	// +listType=atomic
```

Comment:
@apelisse should the linter stop checking listType within a struct labeled as `+structType=atomic`?

Reply:
I did not know `+structType=atomic` was a thing!  None of this seems to be documented.  I'll try removing that and see if it complains.

Reply:
Nope, it still complains:

```
+API rule violation: list_type_missing,k8s.io/apimachinery/pkg/apis/meta/v1,LabelSelector,MatchExpressions
```